cortex/requirements_importer.py

```
@@ -0,0 +1,648 @@
+        path = Path(file_path)
+        if not path.exists():
+            return ParseResult(
+                dependencies=[],
+                dev_dependencies=[],
+                package_manager=PackageManager.BUNDLER,
+                source_file=file_path,
+                errors=[f"File not found: {file_path}"]
+            )
+
+        with open(path, 'r') as f:
+            for line_num, line in enumerate(f, 1):
```

Ruff (0.14.7)

Loop control variable `line_num` not used within loop body

Rename unused `line_num` to `_line_num`

(B007)